net/tailscale/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tailscale
-PKG_VERSION:=1.44.0
+PKG_VERSION:=1.46.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=tailscale-$(PKG_VERSION).tar.gz
```

Comment:
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz

Reply:
resolved in <https://github.com/openwrt/packages/commit/aeedbf9c101e3c2fe653ab9f19a0855514b7bdcb>